### DISCUSSION TITLE
How to edit separating dividers

### DISCUSSION THREAD

**Discussion by soheil-h** [alshedivat/al-folio#2311] [Q&A] [answered]
I'm trying to edit the separating lines that exist in "publications" and "projects" but not sure where. My understanding is that they are in "_base.scss" but I appreciate more information on this.

**Comment by george-gca** [comment, marked as answer]
The easiest way to find this and try in advance where you could add the change is by using [chrome dev tools](https://developer.chrome.com/docs/devtools/css) or [firefox dev tools](https://firefox-source-docs.mozilla.org/devtools-user/). In there you can click in the element and find all the attributes that are set for that element and where are they. Look for example where is set the separating line in the right part of the image (check the `border-top` part):

![image](https://github.com/alshedivat/al-folio/assets/31376482/af960574-e870-4504-befc-cf31b54daea3)

**Reply by soheil-h** [reply]
Thank you, it was very helpful!
